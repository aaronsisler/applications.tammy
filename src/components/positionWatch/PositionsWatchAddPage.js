import React from 'react';
import PositionsWatchAddContainer from 'PositionWatch/PositionsWatchAddContainer';

export default class PositionsWatchAddPage extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div id="positions_watch_add_page">
                <PositionsWatchAddContainer />
            </div>
        );
    }
}
