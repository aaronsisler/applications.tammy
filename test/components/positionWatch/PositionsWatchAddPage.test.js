import React from 'react';
import { shallow } from 'enzyme';
import PositionsWatchAddPage from 'PositionWatch/PositionsWatchAddPage';

describe('PositionsWatchAddPage', () => {
    it('should render PositionsWatchAddPage correctly', () => {
        const wrapper = shallow(<PositionsWatchAddPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
