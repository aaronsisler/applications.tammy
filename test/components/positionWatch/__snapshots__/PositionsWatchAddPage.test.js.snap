// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PositionsWatchAddPage should render PositionsWatchAddPage correctly 1`] = `
<div
  id="positions_watch_add_page"
>
  <Connect(PositionsWatchAddContainer) />
</div>
`;
